k8s/upgrades/0.4.0-0.5.0/oebs_update.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Two queries which you may completely ignore:
1/ Can we use a .go file i.e. logic this in .go? Do you see any significant issues?
2/ Do you see it beneficial to package the entire thing as a Docker image & run it as a K8s JOB. Maybe in future?

Reply:
2/ Definitely a candidate to containerize and run inside a K8s job
1/ Can be written.

In both cases, we can consider doing these if an --  "upgrade k8s job" (with its test container running the bash/go upgrade code)  -- is the one which will perform openebs upgrades. If it is a temporary req b/w 0.4-0.5.x then may not be really worth spending time on.